core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -280,9 +277,9 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           case wf if !WorkflowStatuses.withName(wf.status).isDone => dataAccess.workflowQuery.updateStatus(wf, WorkflowStatuses.Aborted)
         }}
 
-        //Gather the Google groups to remove
-        groupRefs: Set[RawlsGroupRef] = workspaceContext.workspace.accessLevels.values.toSet ++ workspaceContext.workspace.realmACLs.values
-        groupsToRemove <- DBIO.sequence(groupRefs.toSeq.map(groupRef => dataAccess.rawlsGroupQuery.load(groupRef)))
+        //Gather the Google groups to remove, but don't remove project owners group which is used by other workspaces
```

Comment:
Should the project owners group be deleted in the case where the last workspace that references it is deleted? Or is there some other type of entity referencing it?